roles/remediate/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# tasks file for remediations
+
+- name: Remediate the system
+  ansible.builtin.include_tasks: "{{ item }}.yml"
+  loop: "{{ remediation_playbooks }}"
+  loop_control:
+    loop_var: remediation
+  when: remediation in remediation_todo
```

Comment:
if this only works on RHEL 8 then you should add a check to only run if it is RHEL 8 system